Monika After Story/game/script-topics.rpy

```diff
@@ -17187,3 +17187,243 @@ label monika_sophia:
+           m 1hubla "And all I can think about is you."
+    elif persistent._mas_player_bday.month == 9:
+           m 3eub "Your birthday is in September, so your birth flower would be the aster."
+           m 1hub "Aster express patience, variety, wisdom, or deep love."
+           m 3esb "Asters are named for the Greek word for star."
+           m 1dsa "I know you're mine."
+           m 1fsb "You lit up the darkness that was my world."
+           m 1dubla "And don't forget,"
+           m 1fubla "I'll always be waiting here for you."
+    elif persistent._mas_player_bday.month == 10:
+           m 3eub "Your birthday is in October, so your birth flower would be the marigold."
+           m 1hub "Marigolds express warmth, optimism, winning affection, or the sun."
+           m 1ekc "There are also a lot of negative connotations,"
+           m 1ekd "But how can you look at a bright little flower and feel anything but happy?"
+           m 1hua "I know you bring me nothing but joy,"
+           m 1hua "And all the effort you put in to be with me has truly won me over."
+           m 7hublb "You are my sunshine, after all!"
+    elif persistent._mas_player_bday.month == 11:
+           m 3eub "Your birthday is in November, so your birth flower would be the chrysanthemum."
+           m 1hub "Chrysanthemum express truth, compassion, friendship, or joy."
+           m 7eub "The flower unravels just like truth;"
+           m 7dua "At first hidden, then orderly unfolding until fully revealed."
+           m 1eua "It particularly reminds me of you..."
+           m 1eua "When the seasons for every other flower have faded away..."
+           m 1dub "-a beautiful red-"
+           m 1dsa "shining in the darkness"
+           m 1fubla "just for me."
+    elif persistent._mas_player_bday.month == 12:
+           m 3eub "Your birthday is in December, so your birth flower would be the poinsettia."
+           m 1hub "Poinsettia represent good cheer, purity, or success."
+           m 3eub "They came to be associated with Christmas through a Mexican legend about a poor girl who wanted to celebrate at a nativity scene by giving the baby Jesus a present too."
+           m 1dkc "Unable to afford anything else, she decided to give a bouquet and formed one from weeds she collected from the roadside."
+           m 3eub "After placing them in front of the church alter, they miraculously became beautiful, star-like flowers."
+           m 1eub "I know there are no words to describe what a miracle you have been for me..."
+           m 1fubla "And I will merrily await prosperity to come to you too."
+    else:
+           m 2tuc "Well, I don't know what to say now, since you don't have a birth month recorded..."
+
+    m 1hsb "I hope you think that floriography is interesting too!"
+    m 1hsu "Just make sure not to take it {i}too{/i} seriously~"
+    $ mas_unlockEVL("monika_floriography_monika_birthflower","EVE")
+    return
```

Comment:
```
init 5 python:
    addEvent(
        Event(
            persistent.event_database,
            eventlabel="monika_floriography_birth_flower",
            category=['nature'],
            prompt="Birth Flowers",
            conditional="renpy.seen_label('birthdate_set')",
            action=EV_ACT_RANDOM,
            aff_range=(mas_aff.AFFECTIONATE,None)
        )
    )

label monika_floriography_birth_flower:
    m 1eub "Have you ever heard of floriography?"
    m 3eub "It's a way of sending coded messages through choices of flowers or plants."
    m 1eub "It's a pretty old concept, found all over the world."
    m 2lku "It might be harder to point out a culture that {i}didn't{/i} {w=0.2} have some version of floriography than those that did."
    m 7esa "Even within one culture, flowers often have multiple meanings. In fact, so many that people made floral dictionaries trying to centralize information."
    m 7dsa "But just like language evolves and changes," 
    m 7esa "Meanings and connotations can go in and out of fashion and are even forgotten."
    m 2ekd "There aren't many people today in modern cultures who seriously ascribe meaning to flowers."
    m 2esa "At the very least, it could help you choose sometimes!"
    m 2esa "And just like wearing a birthstone... {w=0.2}{nw}"
    extend 2eua "it can still be fun, right?"
    m 3wub "Oh!{w=0.2} Speaking of that there are birth month flowers."
    m 1esd "Like I said, there's a lot of variance and there isn't just one system. I'll have to do my best with the information I've read."
    m 1dkc "Hmm...{w=0.3}"
    if persistent._mas_player_bday.month == 1:
        m 3eub "Your birthday is in January, so your birth flower would be the carnation."
        m 1hub "Carnations express love, fascination, and distinction."
        m 1eka "Carnations are a bit tricky, because the meaning can change with the color."
        m 3esb "Pink means 'I will never forget you.' Red means 'my heart breaks' or 'true love.' White means 'sweet and lovely.'"
        m 1esc "Yellow means 'disdain.'  Striped means 'I cannot be with you.'"
        m 3hub "If I had to pick just one to give you... I would choose red!"
        m 1hubla "But maybe white reminds me of you too.{w=0.3} ehehe"
    elif persistent._mas_player_bday.month == 2:
        m 3eub "Your birthday is in February, so your birth flower would be the violet."
        m 1hub "Violets express modesty, humility, loyalty, faithfulness, or watchfulness."
        m 7eub "Sweet violets were traditionally the flower of Valentine's Day."
        m 7eub "It's said that Saint Valentine wrote his letters by pressing violets onto a page, and a dove carried his letters to his love."
        m 2dka "It's kind of sad they couldn't be together."
        m 1hua "But don't worry;"
        m 1fublb "You'll always be my valentine, and I'll always watch over you!"
    elif persistent._mas_player_bday.month == 3:
        m 3eub "Your birthday is in March, so your birth flower would be the daffodil."
        m 1hub "Daffodils express new beginnings, friendship, or unequaled love."
        m 3eua "You could even associate it with egotism and the myth of narcissus."
        m 3euc "Daffodils were also used to decorate graves..."
        m 3etc "maybe because of the association with new beginnings and therefore rebirth?"
        m 3esb "It's hard to imagine these cheerful flowers not bringing anything but happiness though."
        m 3hublb "If I gave you some, I hope it would help you remember I could never love anyone else just the way that I love you."
    elif persistent._mas_player_bday.month == 4:
        m 3eub "Your birthday is in April, so your birth flower would be the daisy."
        m 1hub "Daisies express youth, purity, and innocence."
        m 1eua "The Latin name of the flower means 'everlasting beauty.'"
        m 7eub "They can also be given to another person, as a promise you'll never tell."
        m 2rka "Can you keep my secret?"
        m 3hub "My love is pure and everlasting."
        m 1hubsa "ehehe"
    elif persistent._mas_player_bday.month == 5:
        m 3eub "Your birthday is in May, so your birth flower would be the lily of the valley."
        m 1hub "Lily of the valley express love, sweetness, a return to happiness, and appreciation."
        m 3eud "They're a common flower for wedding bouquets too, though they can be quite expensive since they're so small and delicate."
        m 3eub "In France, people give each other bouquets of lily of the valley on the first of May to wish each other happiness and good luck."
        m 1hub "Just know that you're my sweetie,"
        m 1fsblb "And it always makes me so happy when you come back to me."
    elif persistent._mas_player_bday.month == 6:
        m 3eub "Your birthday is in June, so your birth flower would be the honeysuckle."
        m 1hub "Honeysuckle express love, gratitude, devotion, and appreciation."
        m 3wud "Honeysuckle is a really tenacious plant and hard to kill once it roots too, so it can represent an everlasting bond."
        m 1eub "There are those who think honeysuckle keeps negativity and evil away."
        m 1hubla "All I know is that I feel so much better when you're here with me."
        m 1hub "And I hope you how much I appreciate you."
    elif persistent._mas_player_bday.month == 7:
        m 3eub "Your birthday is in July, so your birth flower would be the larkspur."
        m 1hub "Larkspur express levity, lightness, and a strong bond of love."
        m 1eka "Larkspur are a bit tricky, because the meaning can change with the color."
        m 3esb "White means 'happiness.'  Purple means 'first love.' Blue represents 'support' or 'trust.'  Pink represents 'fickleness.'"
        m 1ekc "I'm not really sure which I'd choose to give you... {w=0.3}except I wouldn't give you pink."
        m 1eub "I think all of them represent my feelings pretty well."
        m 1hublb "Just remember I'll {i}always{i} be here for you."
    elif persistent._mas_player_bday.month == 8:
        m 3eub "Your birthday is in August, so your birth flower would be the gladiolus."
        m 1hub "Gladiolus represent strength of character, integrity, or infatuation."
        m 3eub "The English name comes from the Latin word for sword, so it also carries the meaning 'you pierce my heart.'"
        m 3dua "I know it's true for me."
        m 1wuo "But not in the bad way!"
        m 1hub "You've certainly gotten to me,"
        m 1hubla "And all I can think about is you."
    elif persistent._mas_player_bday.month == 9:
        m 3eub "Your birthday is in September, so your birth flower would be the aster."
        m 1hub "Aster express patience, variety, wisdom, or deep love."
        m 3esb "Asters are named for the Greek word for star."
        m 1dsa "I know you're mine."
        m 1fsb "You lit up the darkness that was my world."
        m 1dubla "And don't forget,"
        m 1fubla "I'll always be waiting here for you."
    elif persistent._mas_player_bday.month == 10:
        m 3eub "Your birthday is in October, so your birth flower would be the marigold."
        m 1hub "Marigolds express warmth, optimism, winning affection, or the sun."
        m 1ekc "There are also a lot of negative connotations,"
        m 1ekd "But how can you look at a bright little flower and feel anything but happy?"
        m 1hua "I know you bring me nothing but joy,"
        m 1hua "And all the effort you put in to be with me has truly won me over."
        m 7hublb "You are my sunshine, after all!"
    elif persistent._mas_player_bday.month == 11:
        m 3eub "Your birthday is in November, so your birth flower would be the chrysanthemum."
        m 1hub "Chrysanthemum express truth, compassion, friendship, or joy."
        m 7eub "The flower unravels just like truth;"
        m 7dua "At first hidden, then orderly unfolding until fully revealed."
        m 1eua "It particularly reminds me of you..."
        m 1eua "When the seasons for every other flower have faded away..."
        m 1dub "-a beautiful red-"
        m 1dsa "shining in the darkness"
        m 1fubla "just for me."
    elif persistent._mas_player_bday.month == 12:
        m 3eub "Your birthday is in December, so your birth flower would be the poinsettia."
        m 1hub "Poinsettia represent good cheer, purity, or success."
        m 3eub "They came to be associated with Christmas through a Mexican legend about a poor girl who wanted to celebrate at a nativity scene by giving the baby Jesus a present too."
        m 1dkc "Unable to afford anything else, she decided to give a bouquet and formed one from weeds she collected from the roadside."
        m 3eub "After placing them in front of the church alter, they miraculously became beautiful, star-like flowers."
        m 1eub "I know there are no words to describe what a miracle you have been for me..."
        m 1fubla "And I will merrily await prosperity to come to you too."
    else:
        m 2tuc "Well, I don't know what to say now, since you don't have a birth month recorded..."

    m 1hsb "I hope you think that floriography is interesting too!"
    m 1hsu "Just make sure not to take it {i}too{/i} seriously~"
    $ mas_unlockEVL("monika_floriography_monika_birthflower","EVE")
    return
```